boards/airfy-beacon/Makefile.include

```diff
@@ -12,5 +12,5 @@ include $(RIOTBOARD)/Makefile.include.serial
 # this board uses openocd
 include $(RIOTBOARD)/Makefile.include.openocd
 
-# include cortex defaults
-include $(RIOTBOARD)/Makefile.include.cortexm_common
+# export board specific includes to the global includes-listing
```

Comment:
As these two lines are identical to all boards, can't they go into the cpu Makefile?